examples/vision/siamese.py

```diff
@@ -378,23 +382,35 @@ def contrastive_loss(y_true, y_pred):
 ## Visualize results
 """
 
-# Plot accuracy curve
-plt.plot(history.history['accuracy'])
-plt.plot(history.history['val_accuracy'])
-plt.title('Model Accuracy')
-plt.ylabel('accuracy')
-plt.xlabel('epoch')
-plt.legend(['train', 'validation'], loc='upper left')
-plt.show()
 
-# Plot loss curve
-plt.plot(history.history['loss'])
-plt.plot(history.history['val_loss'])
-plt.title('Model Loss')
-plt.ylabel('loss')
-plt.xlabel('epoch')
-plt.legend(['train', 'validation'], loc='upper left')
-plt.show()
+def plt_metric(history, metric, title, has_valid=True):
+    """Plots the given 'metric' from 'history'.
+
+    Arguments:
+        history: history attribute of History object returned from Model.fit.
+        metric: Metric to plot, a string value present as key in 'history'.
+        title: A string to be used as title of plot.
+        has_valid: Boolean, true if valid data was passed to Model.fit
+        	    else false.
+
+    Returns:
+        None.
+    """
```

Comment:
@fchollet I added generated files, i made a slight change i.e. added this function to make reusable history plot code. Please check if it's ok